automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGCEClusterMonitoringSpec.scala

```diff
@@ -59,72 +58,6 @@ class NotebookGCEClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelT
       }
     }
 
-    "should update GCR welder on a cluster" taggedAs Retryable in { billingProject =>
```

Comment:
I guess we lose the ability to test this if removing `welderDockerImage`, right?

Reply:
yea...I mentioned in the description...it's really nice to have the test, but at the same time, I don't think we should complicate api for the purpose of testing 🤔 ...wish there's another way to test this